py/selenium/webdriver/chromium/service.py

```diff
@@ -56,5 +58,15 @@ def __init__(
             **kwargs,
         )
 
+    @property
+    def service_args(self):
+        return self._service_args
+
+    @service_args.setter
+    def service_args(self, value):
+        if not isinstance(value, list):
+            raise TypeError("service args must be a list")
+        self._service_args.extend(value)
```

Comment:
a `@setter` should probably replace the entire sequence, not extend to it - this will create confusing behaviour